### DISCUSSION TITLE
v0.7.4 Release Cut

### DISCUSSION THREAD

**Discussion by ollieanwyll** [danny-avila/LibreChat#3523] [Q&A] [answered]
I’m curious to know if there’s an estimate for cutting a v0.74 release. I’m currently on v0.73 and am contemplating whether to deploy the main branch or wait for an official release. 

Cheers, and amazing work!

**Comment by berry-13** [comment]
> I’m curious to know if there’s an estimate for cutting a v0.74 release. I’m currently on v0.73 and am contemplating whether to deploy the main branch or wait for an official release.
> 
> Cheers, and amazing work!

LibreChat's latest version is `0.7.4-rc1`

**Comment by ollieanwyll** [comment]
Thanks for pointing it out. I did notice the release candidate, but I think
I’ll hold out until the next release, which contains some of the new
features that have been developed over the last few weeks.

**Comment by danny-avila** [comment, marked as answer]
Hey @ollieanwyll 

I answered this concern in this discussion, I'm cutting a release this Tuesday: https://github.com/danny-avila/LibreChat/discussions/3519#discussioncomment-10223239

**Reply by ollieanwyll** [reply]
Thank you @danny-avila and team!
